components/aws/sagemaker/tests/integration_tests/resources/definition/hpo_pipeline.py

```diff
@@ -0,0 +1,65 @@
+import kfp
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+sagemaker_hpo_op = components.load_component_from_file(
+    "../../hyperparameter_tuning/component.yaml"
+)
+
+
+@dsl.pipeline(
+    name="SageMaker HyperParameter Tuning", description="SageMaker HPO job test"
+)
+def hpo_pipeline(
+    region="",
+    algorithm_name="",
+    training_input_mode="",
+    static_parameters="",
+    integer_parameters="",
+    channels="",
+    categorical_parameters="",
+    early_stopping_type="",
+    max_parallel_jobs="",
+    max_num_jobs="",
+    metric_name="",
+    metric_type="",
+    hpo_strategy="",
+    instance_type="",
+    instance_count="",
+    volume_size="",
+    max_run_time="",
+    output_location="",
+    network_isolation="",
+    max_wait_time="",
+    role="",
```

Comment:
do you really have to put these default string `""` ?

Reply:
not necessarily but leaving this as is for now